<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-9HQCIRO" ignored="1" name="Guru99 inetBanking Test" tests="5" failures="3" timestamp="28 Mar 2020 14:37:43 GMT" time="98.889" errors="0">
  <testcase name="addNewCustomer" time="29.254" classname="com.inetBanking.testCases.TC_NewCustomer_003"/>
  <testcase name="loginDDT" time="8.726" classname="com.inetBanking.testCases.TC_loginTest_001"/>
  <testcase name="loginDDT" time="4.23" classname="com.inetBanking.testCases.TC_loginTest_001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_loginTest_001.loginDDT(TC_loginTest_001.java:58)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="4.099" classname="com.inetBanking.testCases.TC_loginTest_001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_loginTest_001.loginDDT(TC_loginTest_001.java:58)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="6.29" classname="com.inetBanking.testCases.TC_loginTest_001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_loginTest_001.loginDDT(TC_loginTest_001.java:58)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginTest" time="0.0" classname="com.inetBanking.testCases.TC_loginTest_001">
    <ignored/>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Guru99 inetBanking Test -->
